homeassistant/components/statistics/sensor.py

```diff
@@ -275,7 +274,9 @@ def _purge_old(self):
         If MaxAge is provided then query will restrict to entries younger then
         current datetime - MaxAge.
         """
+        # pylint: disable=import-outside-toplevel
         from homeassistant.components.recorder.models import States
+        from homeassistant.components.recorder.util import execute, session_scope
```

Comment:
Please put both at the top. We used to only load models later because it required SQLAlchemy to be installed.

Reply:
Can you expand on this, please. It still requires sqlalchemy to be installed and we don't depend on the recorder here.

Reply:
Oh right, my bad. This will be the case once we merge #29494 (which adds recorder as after_dependencies for statistics integration) and #29491 (which installs requirements for after_dependencies)

Reply:
Ok, good!

Reply:
It's merged now

Reply:
@springstan please move these two imports to the top, and add the `"recorder"` integration to `"after_dependencies"` in manifest.json.

Reply:
Sure thing! :) fixed in https://github.com/home-assistant/home-assistant/pull/29223/commits/bf3333532ed1dd60cdf87f3364da62fa3ed03acd